index.html

```diff
@@ -104,9 +108,216 @@ <h2 class="color-heading" id="heading-2">
                 </a>
               </li>
             </ul>
+            <div class="downward arrow">
+              <img src="assets/images/Action.png" alt="" />
+            </div>
           </div>
         </div>
       </div>
     </section>
+
+    <section class="work-section">
+      <div class="opening-heading">
+        <h2 class="work-heading">My Recent Work</h2>
+      </div>
+      <div class="container-box container-grid">
+        <div class="section-container1 section-box item-grid">
+          <ul class="first-work display-flex-1">
+            <li class="project1-image display-flex-width">
+              <img
+                class="p1"
+                src="assets/images/work-project1.png"
+                alt="Tonic Project: Personalized reading selection"
+                rel="noopener noreferrer"
+              />
+            </li>
+            <li class="display-flex-width heading-height">
+              <h3 class="project-heading">
+                Tonic Project: Personalized Reading
+              </h3>
+            </li>
```

Comment:
- [ ] Use `a` a tag that will redirect to the project specifications like live demo link or on GitHub.